src/components/dashboard/DashboardPage.jsx

```diff
@@ -16,6 +16,7 @@ import {
   useSubscriptions,
 } from '../app/data';
 import BudgetExpiryNotification from '../budget-expiry-notification';
+import ExpiredSubscriptionModal from '../course/ExpiredSubscriptionModal';
```

Comment:
[suggestion] I'd recommend avoiding importing modules from `course` module into the `dashboard` module (and `search` module below), as it introduces code coupling between page routes. Could this `ExpiredSubscriptionModal` component be abstracted higher in the component tree so its above the (mostly) independent `course`, `search`, `dashboard` modules?